packages/react/src/components/Search/Search-test.js

```diff
@@ -208,6 +210,12 @@ describe('Search', () => {
         input.simulate('change', eventObject);
         expect(onChange).toHaveBeenCalledWith(eventObject);
       });
+
+      it('should invoke onClear when input value is cleared', () => {
+        wrapper.setProps({ defaultValue: 'test' });
+        wrapper.find('button').simulate('click');
+        expect(onClear).toHaveBeenCalled();
+      });
```

Comment:
it looks like this test is failing

<img width="1160" alt="Screen Shot 2021-10-06 at 1 39 14 AM" src="https://user-images.githubusercontent.com/32556167/136152851-d2b4152a-5c2e-429b-af64-90dc57b13ee6.png">